### PR TITLE
JSON Title Music

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
Changes how title music is parsed from the configuration. All title musics that aren't in `ROUND_START_MUSIC_LIST` require an accompanying JSON providing the following information:
- Name
- Author
- Filepath
- Map (Can be null)
- Rare (Boolean)

ROUND_START_MUSIC_LIST music will simply print the file name.

## Why It's Good For The Game
No more players asking "What song is this?"

## Changelog

:cl:
add: Title music will now display song name and author.
/:cl:

### CODE REVIEW

File: config/title_music/EXAMPLE.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"name" : "Starman",
+	"author" : "David Bowie",
+	"file" : "title_music/sounds/starman.ogg",
+	"map" : false,
+	"rare" : false
+}
```
Comment (ZephyrTFA) [inline config/title_music/EXAMPLE.json:7]: Are json files supposed to have an EOL newline?

Comment (Kapu1178) [reply config/title_music/EXAMPLE.json:7]: I... have no idea.

Comment (ZephyrTFA) [reply config/title_music/EXAMPLE.json:7]: Gonna assume that Byond doesn't give a shit

**Review by ZephyrTFA — APPROVED** [review]

File: config/title_music/EXAMPLE.json (hunk 1 above)
Comment (francinum) [reply config/title_music/EXAMPLE.json:7]: BYOND's json parser is lazy enough to not care.

**Review by francinum — APPROVED** [review]
insert jamiroquai here

**Pull request merged**